taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -17,32 +20,62 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
     const [renderedTranslations, setRenderedTranslations] = useState([]) // Stores state of all tags
     const [mainLangRenderedTranslations, setMainLangRenderedTranslations] = useState([]) // Stores state of main language's tags
     const [openDialog, setOpen] = useState(false); // Used for Dialog component
-    const [newLanguageCode, setNewLanguageCode] = useState(''); // Used for storing new LC from Dialog
-    const [isValidLanguageCode, setisValidLanguageCode] = useState(false); // Used for validating a new LC
+    const [addedLanguageCodes, setAddedLanguageCodes] = useState([]); // Used for storing LC's that are added
+    const [shownTranslationLanguages, setShownTranslationLanguages] = useState([]); // Used for storing LC's that are to be shown
+    const [newLanguageCodes, setNewLanguageCodes] = useState([]); // Used for storing LC's that are to be added to the list of shown LC's 
```

Comment:
For this PR let's focus on just the collapse/expand action. Let's try by using only an array that contains all the expanded languages `const [expandedLanguages, setExpandedLanguages] = useState([])`